pages/database-management/enabling-memgraph-enterprise.mdx

```diff
@@ -8,19 +8,20 @@ description: Learn how to enable Memgraph Enterprise and leverage its advanced c
 The following Memgraph features are only available in Enterprise Edition:
 
 - [Audit log](/database-management/logs#audit-log-enterprise)
-- [Auth system integrations](/database-management/authentication-and-authorization/auth-system-integrations)
+- [Auth system integrations](/database-management/authentication-and-authorization/auth-system-integrations) and [Single Sign-On (SSO)](/database-management/authentication-and-authorization/auth-system-integrations#single-sign-on)
 - [Hiding sensitive information](/database-management/logs#hiding-sensitive-information-enterprise)
 - [SAML integration](/database-management/authentication-and-authorization/auth-system-integrations#saml)
 - [OIDC integration](/database-management/authentication-and-authorization/auth-system-integrations#openid-connect-oidc)
 - [LDAP integration](/database-management/authentication-and-authorization/auth-system-integrations#ldap)
-- [OIDC integration](/database-management/authentication-and-authorization/auth-system-integrations#oidc)
 - [Monitoring via HTTP server](/database-management/monitoring#monitoring-via-http-server-enterprise)
```

Comment:
this anchor doesn't exist... it should be #metrics-tracking-via-http-server-enterprise-edition